### DISCUSSION TITLE
Next release ? Opentofu Registry ?

### DISCUSSION THREAD

**Discussion by lukry59** [terraform-coop/terraform-provider-foreman#153] [General]
Hello,

I'd like to know when the next release is planned?

Also, do you plan to publish the provider on the OpenTofu registry?

Regards,

**Comment by lhw** [comment]
In general i am waiting for someone to approve the build-system PR. After that I would create a new release. A opentofu release would be fine to. I'll check whats required for that.

**Reply by lhw** [reply]
It's already in the opentofu apparently https://github.com/opentofu/registry/issues/348#issuecomment-2022991663

**Reply by bitkeks** [reply]
It's here: https://github.com/opentofu/registry/blob/39d9529a076757f02fa69597138ff9960c43ce28/providers/t/terraform-coop/foreman.json

Though there are multiple versions:
* [providers/j/jgrancell/foreman.json](https://github.com/opentofu/registry/blob/39d9529a076757f02fa69597138ff9960c43ce28/providers/j/jgrancell/foreman.json)
* [providers/h/HanseMerkur/foreman.json](https://github.com/opentofu/registry/blob/39d9529a076757f02fa69597138ff9960c43ce28/providers/h/HanseMerkur/foreman.json)
* [providers/t/terraform-coop/foreman.json](https://github.com/opentofu/registry/blob/39d9529a076757f02fa69597138ff9960c43ce28/providers/t/terraform-coop/foreman.json)

**Comment by bitkeks** [comment]
> I'd like to know when the next release is planned?

Hi, we're currently working on it. After all the PRs were merged, we did another testing round and are now investigating last bugs in the Katello stuff. After that, a new release is planned. It will probably be a `0.7` due to multiple additions for Katello and a refactoring of the logging code (https://github.com/terraform-coop/terraform-provider-foreman/pull/146).
